components/esp-tls/esp_tls.h

```diff
@@ -227,7 +243,13 @@ typedef struct esp_tls_cfg_server {
     unsigned int serverkey_password_len;        /*!< String length of the password pointed to by
                                                      serverkey_password */
 
+#if defined(CONFIG_ESP_TLS_USING_MBEDTLS) && defined(CONFIG_ESP_TLS_SERVER_SESSION_TICKETS)
+    esp_tls_session_ticket_ctx_t ticket_ctx;                          /*!< Session ticket generation context */
+#endif
 } esp_tls_cfg_server_t;
+
+int esp_tls_cfg_server_init(esp_tls_cfg_server_t * cfg);
```

Comment:
As this is a public API, also add appropriate docs here.

Reply:
I have renamed the function to reflect that only the session ticket context is initialized.
Additionally one does not need to call the function if one does not want to use session tickets.
This way the changes should be compatible with the old API.